db/external_sst_file_ingestion_job.cc

```diff
@@ -226,7 +226,8 @@ Status ExternalSstFileIngestionJob::Prepare(
             &generated_checksum_func_name,
             ingestion_options_.verify_checksums_readahead_size,
             db_options_.allow_mmap_reads, io_tracer_,
-            db_options_.rate_limiter.get(), ro);
+            db_options_.rate_limiter.get(), ro, nullptr /* stats */,
```

Comment:
`db_options_.stats`, otherwise we'll have the same problem next time

Reply:
I don't think we have external sst ingestion read stats yet but will do this fix in advance!

Reply:
Fixed